### PR TITLE
fix(breakout-rooms) Improve breakout rooms

### PR DESCRIPTION
**Pull request by robertpin**
Don't show more menu on the main room

Fix join room on mobile web

When moving to room reset breakout rooms state to avoid showing incorrect data before the response from prosody is received

### CODE REVIEW

File: react/features/breakout-rooms/actions.js [hunk 1]
```diff
@@ -170,6 +171,11 @@ export function moveToRoom(roomId?: string) {
             _roomId.domain = domainParts.join('@');
         }
 
+        dispatch({
+            type: UPDATE_BREAKOUT_ROOMS,
+            rooms: {}
```
Comment (saghul) [inline react/features/breakout-rooms/actions.js:176]: reset the counter too

Comment (saghul) [reply react/features/breakout-rooms/actions.js:176]: maybe we can create an internal action _RESET_BREAKOUT_ROOMS` and apply the default state in the reducer. This way we make it future-proof, should we add more state in the future.

**Review by saghul — DISMISSED** [review]
LGTM, but alas this is going to clash with my PR, so you're going to need to rebase.

File: react/features/breakout-rooms/actionTypes.js [hunk 2]
```diff
@@ -10,3 +10,8 @@ export const UPDATE_BREAKOUT_ROOMS = 'UPDATE_BREAKOUT_ROOMS';
  * The type of (redux) action to update the room counter locally.
  */
 export const _UPDATE_ROOM_COUNTER = '_UPDATE_ROOM_COUNTER';
+
+/**
+ * The type of (redux) action to reset the breakout rooms data.
+ */
+export const _RESET_BREAKOUT_ROOMS = '_RESET_BREAKOUT_ROOMS';
```
Comment (saghul) [inline react/features/breakout-rooms/actionTypes.js:17]: alpha sort please

File: react/features/breakout-rooms/actions.js [hunk 3]
```diff
@@ -15,7 +15,7 @@ import { setAudioMuted, setVideoMuted } from '../base/media';
 import { getRemoteParticipants } from '../base/participants';
 import { clearNotifications } from '../notifications';
 
-import { _UPDATE_ROOM_COUNTER } from './actionTypes';
+import { _UPDATE_ROOM_COUNTER, _RESET_BREAKOUT_ROOMS } from './actionTypes';
```
Comment (saghul) [inline react/features/breakout-rooms/actions.js:18]: alpha sort pl

File: react/features/breakout-rooms/reducer.js [hunk 4]
```diff
@@ -4,7 +4,8 @@ import { ReducerRegistry } from '../base/redux';
 
 import {
     _UPDATE_ROOM_COUNTER,
-    UPDATE_BREAKOUT_ROOMS
+    UPDATE_BREAKOUT_ROOMS,
+    _RESET_BREAKOUT_ROOMS
```
Comment (saghul) [inline react/features/breakout-rooms/reducer.js:8]: group the ones starting with _ at the top, alpha sorted

**Review by saghul — APPROVED** [review]

**Comment by rcidt** [comment]
@saghul how do you enable breakout rooms feature on mobile?

**Comment by saghul** [comment]
It's enabled by default, if the deployment supports it.

**Pull request merged**